extensions/amp-story/1.0/amp-story.js

```diff
@@ -1312,6 +1313,20 @@ export class AmpStory extends AMP.BaseElement {
     );
   }
 
+  /**
+   * Adds an empty alt tag to amp-img elements if not present.
+   * Prevents screen readers from announcing the img src value.
+   * @private
+   */
+  initializeImgAltTags_() {
```

Comment:
Could we do this at the `amp-story-page.js` level? this we have a `getMediaBySelector` method there, maybe we could use that / extend it to find the `amp-img` at the right time


https://github.com/ampproject/amphtml/blob/984f380f9e6f9f2c9fed07956fc69c34bf15a8e2/extensions/amp-story/1.0/amp-story-page.js#L651

Reply:
Doing this at the `amp-story-page.js` level sounds good.

As we discussed, this needs to happen during the buildCallback so that the `amp-image` and inner `img`'s attributes are synced.